.github/workflows/bindings-python.yml

```diff
@@ -111,6 +112,12 @@ jobs:
           sudo apt-get update
           sudo apt-get install libudev-dev libusb-1.0-0-dev
 
-      - name: Run Tox
+      - name: Run linter for examples
+        if: ${{ startsWith(matrix.os, 'ubuntu-latest') }}
```

Comment:
Any specific reason for this `if`?

Reply:
Just a matter of saving time. We don't need to run `pylint` on Windows because it will give the same results.

Reply:
Then we also don't need to run it on macos?

Reply:
Lol ignore my comment, brainfart